src/routes/ImportRoute.js

```diff
@@ -82,36 +82,43 @@ class ImportRoute extends Component {
     );
   }
 
-  handleMnemonicChange = e => {
+  handleMnemonicOrWifChange = e => {
     this.setState({
-      mnemonic: e.target.value,
+      mnemonicOrWif: e.target.value,
     });
   };
 
   handleContinueClick = async e => {
     e.preventDefault();
 
-    if (this.isMnemonicValid()) {
-      const result = await this.props.tryCreateWalletByMnemonic(
-        this.state.mnemonic
+    let result;
+
+    if (this.isMnemonic()) {
+      result = await this.props.tryCreateWalletByMnemonic(
+        this.state.mnemonicOrWif
+      );
+    } else {
+      result = await this.props.tryCreateWalletByPrivateKey(
```

Comment:
We can ensure that value is a private key using https://github.com/SnaxFoundation/snaxjs-ecc#isvalidprivate
Otherwise we may show an error